examples/custom-interactions/view.js

```diff
@@ -21,7 +21,7 @@
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
 import autobind from 'react-autobind';
-import {InteractiveMap, ScatterplotOverlay} from 'react-map-gl';
+import {InteractiveMap, ScatterplotOverlay, Marker} from 'react-map-gl';
```

Comment:
Putting the Marker in this example is fine for now, but we might want to consider how we organize the examples, e.g:
- One example with all the overlays. 
- One example with markers/other react component etc.
- This particular example was initially created to demo breaking mapbox-gl's 60 degree pitch limit.